jest/setup.js

```diff
@@ -18,14 +18,17 @@
 
 /* @flow */
 
+import { useState } from 'react';
 import { NativeModules } from 'react-native';
 
-NativeModules.RNDarkMode = {
-  initialMode: 'light',
-  supportsDarkMode: true,
-  addListener: jest.fn(),
-  removeListeners: jest.fn(),
-};
+jest.mock('react', () => ({
```

Comment:
We mock hooks here, as it helps with mocking of the RN `Appearance` module that is using hooks